src/Dropdown/Dropdown.js

```diff
@@ -289,22 +295,18 @@ class Dropdown extends Util.mixin(BindMixin) {
     return scrollContainer;
   }
 
-  getSelectedHtml(id, items) {
-    const obj = Util.find(items, function(item) {
-      return item.id === id;
-    });
-
-    if (obj != null) {
-      return obj.selectedHtml || obj.html;
-    }
-
-    return null;
-  }
-
   getSelectedID() {
     return this.props.persistentID || this.state.selectedID;
   }
 
+  getSelectedItem() {
+    return (
+      this.props.items.find(
+        item => item.id && item.id === this.getSelectedID()
+      ) || null
```

Comment:
Will `undefined` be a problem here? `find` will return `undefined` unless it finds something.